File: Open_tk_triangle/test/Form1.vb
Public Class Form1

    Dim myPen As Pen
    Dim myPoints As Point() =
    {
                New Point(60, 50),
                New Point(60, 100),
                New Point(10, 100),
                New Point(10, 200),
                New Point(60, 200),
                New Point(60, 250),
                New Point(360, 250),
                New Point(360, 200),
                New Point(410, 200),
                New Point(410, 100),
                New Point(360, 100),
                New Point(360, 50),
                New Point(60, 50)
    }

    Private Sub Form1_Load(sender As Object, e As EventArgs) Handles MyBase.Load

    End Sub

    Private Sub Form1_Paint(ByVal sender As Object, ByVal e As System.Windows.Forms.PaintEventArgs) Handles MyBase.Paint
        myPen = New Pen(Drawing.Color.Blue, 5)
        Dim myGraphics As Graphics = Me.CreateGraphics
        myGraphics.DrawPolygon(myPen, myPoints)
    End Sub
End Class

File: Open_tk_triangle/lab4/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace lab4
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e)
        {
        }

        private void Form1_Paint(object sender, PaintEventArgs e)
        {
            Pen p = new Pen(Color.Black);

            e.Graphics.DrawArc(p, 350, 200, 300, 300, 262, 198);
            e.Graphics.DrawArc(p, 200, 165, 350, 365, 306, 110);

        }
    }
}
